container/README.md

```diff
@@ -0,0 +1,39 @@
+# Modifying the default HiveMQ K8s Image
+Security is important [[1](https://snyk.io/blog/10-kubernetes-security-context-settings-you-should-understand)] when running containers in Kubernetes. 
+For this reason we provide the documentation on how to modify the HiveMQ image without increasing its size.
+
+## Build the image
+
+- Get the latest version from the docker registry https://hub.docker.com/r/hivemq/hivemq4/tags?page=1&name=k8s
+ at the time of writing was `hivemq/hivemq4:k8s-4.8.3`.
```

Comment:
would do something like: 
Get the latest version from the docker registry https://hub.docker.com/r/hivemq/hivemq4/tags?page=1&name=k8s, example: 'hivemq/hivemq4:k8s-\<version\>'